cogs/helpers.py

```diff
@@ -38,18 +36,61 @@
 import re
 import math
 import time
-import os
 import datetime
+import pickle
+import feedparser
 from .utils.paginator import Pages
 from .utils.requests import fetch
 
 MCGILL_EXAM_URL = "https://www.mcgill.ca/exams/dates"
+CFIA_FEED_URL = "http://inspection.gc.ca/eng/1388422350443/1388422374046.xml"
 
 
 class Helpers():
     def __init__(self, bot):
         self.bot = bot
 
+    async def cfia_rss(self):
+        # Written by @jidicula
+        """
+        Co-routine that periodically checks the CFIA Health Hazard Alerts RSS
+         feed for updates.
+        """
+        await self.bot.wait_until_ready()
+        while not self.bot.is_closed():
+            food_channel = utils.get(
+                self.bot.get_guild(self.bot.config.server_id).text_channels,
+                name="food")
+            newest_recall = feedparser.parse(CFIA_FEED_URL)['entries'][0]
+            newest_recall_id = newest_recall['id']
+            try:
+                id_unpickle = open("pickles/recall_tag.obj", 'rb')
```

Comment:
will I have to create the pickles directory? if so, can there be a check to create the pickles directory if needed?

Reply:
I created the pickles directory and pushed the object actually. This isn't a great longterm solution – I can add an os.mkdir with a try/except.

Reply:
sure, that would be good. and probably add pickles/* to gitignore